package.json

```diff
@@ -52,42 +52,42 @@
     "preversion": "npm-run-all --npm-path yarn --parallel check-audit format:check lint:* test:coverage",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "version": "node version.js && git add -A CHANGELOG.md",
+    "version": "curl https://raw.githubusercontent.com/marp-team/marp/master/version.js | node && git add -A CHANGELOG.md",
```

Comment:
We also have the updated `version` script for using marp-team common script hosted at [marp-team/marp](https://github.com/marp-team/marp).